Single-Sign-on/faqs.md

```diff
@@ -0,0 +1,165 @@
+
+### Q: What SSO flows are supported by Docker?
+
+Docker currently supports Service Provider Initiated (SP-initiated) SSO flow. This means, users must sign in to Docker Hub or Docker Desktop to initiate the SSO authentication process.
+
+### Q: Can I enable SSO in all organizations?
+
+You can enable SSO on organizations that are part of the Docker Business subscription.
+
+### Q: We currently have a Docker Team subscription. How do we enable SSO?
+
+Docker SSO is available with a Docker Business subscription. To enable SSO, you must first upgrade your subscription to a Docker Business subscription. To learn how to upgrade your existing account, see [Upgrade your subscription](/subscription/upgrade/){:target="blank" rel="noopener" class=""}.
+
+### Q: Where can I find detailed instructions on how to configure Docker SSO?
+
+For step by step instructions on how to configure Docker SSO, see [SSO docs](/single-sign-on).
+
+### Q: Is it possible to use more than one IdP with Docker SSO?
+
+No. You can only configure Docker SSO to work with a single IdP. A domain can only be associated with a single IdP.Docker currently supports Azure AD and identity providers that support SAML 2.0
+
+### Q: Is it possible to change my identity provider after configuring SSO?
+
+Yes. You must delete your existing IdP configuration in Docker Hub and follow the instructions in the [SSO docs](/single-sign-on) to configure SSO using your IdP.
+
+###  Q: Is Docker SSO available for all paid subscriptions?
+
+Docker SSO is only available with the Docker Business subscription. [Upgrade](/subscription/upgrade/){:target="blank" rel="noopener" class=""} your existing subscription to start using Docker SSO.
+
+### Q: Does Docker SSO support multi-factor authentication (MFA)?
+
+When SSO is being used by an organization, MFA is determined at the idP level and not the Docker system.
+
+### Q: How does service accounts work with SSO?
+
+Service accounts work like any other user when SSO is turned on. If the service account is using an email for a domain with SSO turned on, it needs a [PAT](/docker-hub/access-tokens/) for CLI and API usage.
+
+
+## Configuration
+
+### Q: What information do I need from my Identity providers to configure SSO?
+
+To enable SSO in Docker, you need the following from your IdP:
+
+* **SAML 2.0**: Entity ID, ACS URL, Single Logout URL and Certificate Download URL
+* **Azure AD**: Client ID, Client Secret, AD Domain
+
+### Q: Is DNS verification required to enable SSO?
+
+Yes. You must verify a domain before using it with an SSO connection.
+
+### Q: Does Docker SSO support authenticating through the command line?
+
+Yes. When SSO is enabled, you can access the Docker CLI through Personal Access Tokens (PATs).  Each user must create a PAT to access the CLI. To learn how to create a PAT, see [Managing access tokens](/docker-hub/access-tokens/).
```

Comment:
Can we add: Before we transition to PATs. CLI users will continue to be able to log in using their personal credentials until early next year to mitigate risk of interrupting CI/CD pipelines.

Reply:
Updated @izzychu